mybatis-spring-boot-test-autoconfigure/pom.xml

```diff
@@ -30,18 +30,37 @@
   <name>mybatis-spring-boot-test-autoconfigure</name>
   <dependencies>
     <dependency>
-      <groupId>org.mybatis.spring.boot</groupId>
-      <artifactId>mybatis-spring-boot-starter</artifactId>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-test-autoconfigure</artifactId>
     </dependency>
     <dependency>
-      <groupId>org.springframework.boot</groupId>
-      <artifactId>spring-boot-starter-test</artifactId>
-      <optional>true</optional>
+      <groupId>org.springframework</groupId>
+      <artifactId>spring-test</artifactId>
+      <exclusions>
+        <exclusion>
+          <groupId>commons-logging</groupId>
+          <artifactId>commons-logging</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework</groupId>
+      <artifactId>spring-tx</artifactId>
     </dependency>
     <dependency>
       <groupId>com.h2database</groupId>
       <artifactId>h2</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.mybatis.spring.boot</groupId>
```

Comment:
It is for testing. If does not used `mybatis-spring-boot-starter`, we need to add following artifacts as `test` scope.

* mybatis-spring-boot-autoconfigure
* mybatis
* mybatis-spring
* spring-jdbc
* spring-boot-starter

Is problem to use `mybatis-spring-boot-starter` as `test` scope ?